apps/developer-hub/content/docs/price-feeds/pro/index.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Pyth Pro"
+description: >-
+  Pyth Pro is a low latency, highly customizable price oracle.
+full: false
+index: true
+---
+
+import { ChartBar, Key, Lightning } from "@phosphor-icons/react/dist/ssr";
+import { IntegrationCard } from "../../../../src/components/IntegrationCard";
+
+<InfoBox variant="info">Pyth Pro was previously known as Pyth Lazer.</InfoBox>
+
+Pyth Pro delivers customizable, enterprise-grade price data directly from first-party publishers.
+Subscribers can configure their price feeds, update schedules, and usage rights for display or redistribution.
+The service is delivered through standard APIs for seamless integration.
+
+---
+
+<div className="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-3">
```

Comment:
I think we should avoid tailwind and use sass for anything we write -- tailwind is used internally by fumadocs but all of this would break if we were to e.g. move to a different markdown renderer.

We should probably see if we can configure the fumadocs built-in tailwind to not look at the `content` folder...